src/app/app.component.ts

```diff
@@ -18,6 +19,23 @@ const HEROES: Hero[] = [
   { id: 20, name: 'Tornado' }
 ];
 
+@Injectable()
+export class Profile {
+  profileService = new ProfileService();
```

Comment:
Instead of creating new objects a constructor should be used for services to allow for Angular dependency injection and unit testing. e.g.

```
export class Profile {
    constructor(
        private profileService: ProfileService
    ) { }
```

See more information [typescript constructor injection](https://angular.io/guide/dependency-injection)